src/en/space-station-14/mapping.md

```diff
@@ -239,6 +245,9 @@ Use this to name specific doors on your map or give flavor to warning signs and
 - If you want to test your cameras view, save your map then run mapinit. You can then view the camera monitor and check their view, make sure to run togglelight in console to get the actual view.
 - For entertainment cameras, follow the same process as above but with cameras marked (entertainment).
 
+## Telecoms
+Stations require a telecommunication server in order for headsets and radios to work correctly. Make sure to place a `TelecomServerFilled` somewhere on your map and sign post it and put a station beacon in there.
+
```

Comment:
Don't most mappers make separate telecom servers for each channel? However, I think it's fine to include one with all channels. I suggest mentioning it.

Reply:
its used on tiny maps that dont have room for full comms

Reply:
May as well mention the way its done now too.

Reply:
> May as well mention the way its done now too.

All news to me but yeah I'll expand on the telecoms section about inserting encryption keys. Was just going off what was in my entity spawn menu at the time. Got some troubleshooting things to include too for map renderer.